### PR TITLE
[Logs] Add Elastic integration docs

### PR DESCRIPTION
**Pull request by angelampcosta**

### CODE REVIEW

File: content/logs/get-started/enable-destinations/elastic/index.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    "name":"<public domain>",
+    "destination_conf": "https://<public domain>:<public port>?header_<secret_header>=<secret_value>",
+    "dataset": "http_requests",
+    "logpull_options": "fields=RayID,EdgeStartTimestamp&timestamps=rfc3339"
+}'
+```
+
+## Enable the Integration in Elastic
+
+Once the Logpush job is configured, follow Elastics instructions for [setting up the Integration](https://docs.elastic.co/integrations/cloudflare) in the Elastic app.
```
Comment (jsells1) [inline content/logs/get-started/enable-destinations/elastic/index.md]: this is the correct Elastic doc link: https://docs.elastic.co/integrations/cloudflare_logpush

Comment (angelampcosta) [reply content/logs/get-started/enable-destinations/elastic/index.md]: Thanks @jsells1 I corrected the link.

File: content/logs/get-started/enable-destinations/elastic/index.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+title: Enable Elastic
+pcx_content_type: how-to
+weight: 59
```
Comment (marciocloudflare) [inline content/logs/get-started/enable-destinations/elastic/index.md:4]: Any reason for the `59` number? If you don't really need this page to be in a specific place you can just remove it.

File: content/logs/get-started/enable-destinations/elastic/index.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+layout: single
+meta:
+  title: Enable Logpush to Elastic
+---
+
+# Enable Logpush to Elastic
+
+Push your Cloudflare logs to Elastic for instant visibility and insights. Enabling this integration with Elastic comes with a predefined dashboard to view all of your Cloudflare Observability and Security data with ease.
```
Comment (marciocloudflare) [inline content/logs/get-started/enable-destinations/elastic/index.md]:
```suggestion
Push your Cloudflare logs to Elastic for instant visibility and insights. Enabling this integration with Elastic comes with a predefined dashboard to view all of your Cloudflare observability and security data with ease.
```

File: content/logs/get-started/enable-destinations/elastic/index.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+The Cloudflare Logpush integration can be used in three different modes to collect data:
+
+- HTTP Endpoint mode - Cloudflare pushes logs directly to an HTTP endpoint hosted by your Elastic Agent.
+- AWS S3 polling mode - Cloudflare writes data to S3 and Elastic Agent polls the S3 bucket by listing its contents and reading new files.
+- AWS S3 SQS mode - Cloudflare writes data to S3, S3 pushes a new object notification to SQS, Elastic Agent receives the notification from SQS, and then reads the S3 object. Multiple Agents can be used in this mode.
```
Comment (marciocloudflare) [inline content/logs/get-started/enable-destinations/elastic/index.md]:
```suggestion
- **HTTP Endpoint mode** - Cloudflare pushes logs directly to an HTTP endpoint hosted by your Elastic Agent.
- **AWS S3 polling mode** - Cloudflare writes data to S3, and Elastic Agent polls the S3 bucket by listing its contents and reading new files.
- **AWS S3 SQS mode** - Cloudflare writes data to S3, S3 pushes a new object notification to SQS, Elastic Agent receives the notification from SQS, and then reads the S3 object. Multiple Agents can be used in this mode.
```

Comment (marciocloudflare) [reply content/logs/get-started/enable-destinations/elastic/index.md]: Also, not sure what to use here but I think that in the sentence `Cloudflare writes data to S3, and Elastic Agent` we need something like `the Elastic Agent` or `an Elastic Agent`

File: content/logs/get-started/enable-destinations/elastic/index.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+}'
+```
+
+## Enable the Integration in Elastic
+
+Once the Logpush job is configured, follow Elastics instructions for [setting up the Integration](https://docs.elastic.co/integrations/cloudflare_logpush) in the Elastic app.
+
+## View Dashboards
+
+Login to your [Elastic account](https://www.elastic.co/) to view pre built dashboards and configure alerts.
```
Comment (marciocloudflare) [inline content/logs/get-started/enable-destinations/elastic/index.md]:
```suggestion
Log in to your [Elastic account](https://www.elastic.co/) to view prebuilt dashboards and configure alerts.
```

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**